tests/test_url.py

```diff
@@ -647,6 +647,14 @@ def test_with_query():
     assert str(url.with_query({'a': '1'})) == 'http://example.com/?a=1'
 
 
+def test_with_query_kwargs():
+    url = URL('http://example.com')
+    assert str(url.with_query(
+        {'a': '2', 'b': '4'}, a='1')) == 'http://example.com/?a=1'
+    assert 'query2=1' in str(url.with_query(query='1', query2='1'))
```

Comment:
Change checks to `str(...) == '...'` form.
`in` is too weak and not obvious comparing to all other tests for the library.